extension.js

```diff
@@ -888,16 +888,36 @@ const EasyScreenCastIndicator = GObject.registerClass({
      * @private
      */
     _replaceStdIndicator(OPTtemp) {
-        if (Main.panel.statusArea['aggregateMenu']._screencast === undefined) {
+        if (Main.panel.statusArea === undefined) {
+            Lib.TalkativeLog('-*-no Main.panel.statusArea found');
+            return;
+        }
+
+        var stdMenu;
+
+        if (Main.panel.statusArea.quickSettings) {
+            // since gnome 43: aggregateMenu -> quickSettings
+            stdMenu = Main.panel.statusArea.quickSettings;
+        } else {
+            stdMenu = Main.panel.statusArea.aggregateMenu;
+        }
+
+        if (stdMenu === undefined) {
+            Lib.TalkativeLog('-*-no quickSettings or aggregateMenu in Main.panel.statusArea');
+            return;
+        }
+        var indicator = stdMenu._screencast || stdMenu._remoteAccess;
```

Comment:
_screencast is also history with 3.38 already: https://gitlab.gnome.org/GNOME/gnome-shell/-/blob/3.38.3/js/ui/panel.js#L738
So - we only need to check for _remoteAccess indicator - but the indicator is called `_recordingIndicator`: https://gitlab.gnome.org/GNOME/gnome-shell/-/blob/3.38.3/js/ui/status/remoteAccess.js#L72 and not just `_indicator`. Only with 43, it is `_indicator`.